dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantCollectionOptions.cs

```diff
@@ -32,4 +32,9 @@ public sealed class QdrantCollectionOptions
     /// Gets or sets the default embedding generator for vector properties in this collection.
     /// </summary>
     public IEmbeddingGenerator? EmbeddingGenerator { get; set; }
+
+    /// <summary>
+    /// Gets or sets value indicating whether the client should be disposed after the vector store is disposed.
+    /// </summary>
+    public bool OwnsClient { get; set; } = QdrantVectorStoreOptions.Default.OwnsClient;
```

Comment:
Maybe `OwnsQdrantClient` to make it a bit more explicit?

Reply:
It's `QdrantCollectionOptions`, I would not expect any other client than `Qdrant`. And we have did such a great job of simplifying the names elsewhere that I would just keep it simple